.github/workflows/build.yml

```diff
@@ -19,6 +19,70 @@ jobs:
 
     name: ${{ matrix.name }}
 
+    env:
+      ARTIFACTS_DIR: exported-artifacts
+
+    container:
+      image: quay.io/centos/centos:${{ matrix.container-name }}
+
     steps:
-      - name: Sanity Check
-        run: echo Hello world
+      - name: Prepare CentOS Stream 8 environment
+        if: ${{ matrix.shortcut == 'cs8' }}
+        run: |
+          # Install oVirt repositories
+          dnf copr enable -y ovirt/ovirt-master-snapshot centos-stream-8
+          dnf install -y ovirt-release-master
+          # Configure CS8 repositories
+          dnf config-manager --enable powertools
+          dnf module enable -y pki-deps javapackages-tools
+      - name: Install required packages
```

Comment:
Most of those dependencies should already be in if you use oVirt buildcontainer, please minimize the list